src/main/java/logisticspipes/recipes/RecipeManager.java

```diff
@@ -3156,30 +3154,17 @@ public static void loadRecipes(ICraftingParts parts) {
                     });
         }
 
-        for (int i = 0; i < 1000; i++) {
-            LogisticsModule module = LogisticsPipes.ModuleItem.getModuleForItem(
-                    new ItemStack(LogisticsPipes.ModuleItem, 1, i), null, null, null);
-            if (module != null) {
-                NBTTagCompound nbt = new NBTTagCompound();
-                boolean force = false;
-                try {
-                    module.writeToNBT(nbt);
-                } catch (Exception e) {
-                    force = true;
-                }
-                if (!nbt.equals(new NBTTagCompound()) || force) {
-                    RecipeManager.craftingManager.addShapelessResetRecipe(LogisticsPipes.ModuleItem, i);
-                }
-            }
-        }
-
-        for (int i = 1; i < 17; i++) {
-            RecipeManager.craftingManager.addOrdererRecipe(
-                    new ItemStack(LogisticsPipes.LogisticsRemoteOrderer, 1, i),
-                    dyes[i - 1],
-                    new ItemStack(LogisticsPipes.LogisticsRemoteOrderer, 1, -1));
-            RecipeManager.craftingManager.addShapelessResetRecipe(LogisticsPipes.LogisticsRemoteOrderer, i);
-        }
-        RecipeManager.craftingManager.addShapelessResetRecipe(LogisticsPipes.LogisticsRemoteOrderer, 0);
+		for (int moduleId : LogisticsPipes.ModuleItem.getRegisteredModulesIDs()) {
+			RecipeManager.craftingManager.addShapelessResetRecipe(LogisticsPipes.ModuleItem, moduleId);
+		}
+
+//        for (int i = 1; i < 17; i++) {
+//            RecipeManager.craftingManager.addOrdererRecipe(
+//                    new ItemStack(LogisticsPipes.LogisticsRemoteOrderer, 1, i),
+//                    dyes[i - 1],
+//                    new ItemStack(LogisticsPipes.LogisticsRemoteOrderer, 1, -1));
+//            RecipeManager.craftingManager.addShapelessResetRecipe(LogisticsPipes.LogisticsRemoteOrderer, i);
+//        }
+//        RecipeManager.craftingManager.addShapelessResetRecipe(LogisticsPipes.LogisticsRemoteOrderer, 0);
```

Comment:
Why is this commented out?

Reply:
The whole `addShapelessResetRecipe` was commented out by gleese earlier. But he/she accidentally commented out the stuff, which was needed to stay the same. Instead of commenting addShapelessRecipe, I only commented the part, which still should not be used in the pack due to the ability to reset remote orderers with chemical bath.

Reply:
Then remove that comment completely, we have git history, no need to keep it
```suggestion
```